client/src/pages/UpdateCohort.js

```diff
@@ -0,0 +1,152 @@
+import { useState } from "react";
+import { Button } from "../component/Button";
+import { InputeField } from "../component/InputeField";
+import { useParams } from "react-router-dom";
+
+export const UpdateCohort = () => {
+	const [formInput, setFormInput] = useState({
+		name: "",
+		start_date: "",
+		m_1: "",
+		m_2: "",
+		m_3: "",
+		m_4: "",
+		m_5: "",
+		m_6: "",
+		m_7: "",
+		m_8: "",
+	});
+	const [updateMessage, setUpdateMessage] = useState("");
+	const { id } = useParams();
+
+	const updateCohort = async (input) => {
+		console.log(input);
+		try {
+			const response = await fetch(`/api/cohorts/${id}`, {
+				method: "PUT",
+				headers: {
+					"Content-Type": "application/json",
+				},
+				body: JSON.stringify(input),
+			});
+
+			const data = await response.json();
+			console.log(data);
```

Comment:
please remove console.logs other than possibly for errors which should be console.error()